src/libs/Network/SequentialQueue.ts

```diff
@@ -102,16 +112,24 @@ function process(): Promise<void> {
 function flush() {
     // When the queue is paused, return early. This will keep an requests in the queue and they will get flushed again when the queue is unpaused
     if (isQueuePaused) {
+        console.debug('[SequentialQueue] Unable to flush. Queue is paused.');
+        return;
+    }
+
+    if (isSequentialQueueRunning) {
+        console.debug('[SequentialQueue] Unable to flush. Queue is already running.');
         return;
     }
 
-    if (isSequentialQueueRunning || PersistedRequests.getAll().length === 0) {
+    if (PersistedRequests.getAll().length === 0) {
+        console.debug('[SequentialQueue] Unable to flush. No requests to process.');
```

Comment:
I'd be game to changing all of these to `Log.info()`. It would have been really helpful for debugging Doza's case where we had to look at logs from the previous day.